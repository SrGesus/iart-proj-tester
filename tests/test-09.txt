FE	VD	FE	VE	VB
VE	BC	LV	FE	LV
FE	BB	BC	BC	BE
FE	VD	FD	BD	LV
FC	LH	LH	VE	FD