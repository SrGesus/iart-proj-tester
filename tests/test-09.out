FD	VE	FB	VB	VE
VB	BE	LV	FC	LV
FC	BD	BC	BB	BE
FD	VC	FD	BE	LV
FD	LH	LH	VC	FC
